README.md

```diff
@@ -6,20 +6,10 @@
 4. Restart Home Assistant
 5. Go to Grocy-Wrench icon-Manage API keys-Add
 6. Copy resulting API key and Grocy URL and input this in configuration.yaml:
+7. Choose:
+   - Add `Grocy:` to your HA configuration.
```

Comment:
I guess this should be `grocy:`

Reply:
Yes, sorry this is the component name and not showing name.